week-05/README.md

```diff
@@ -15,53 +15,56 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Textbook Chapter 5, Problem 1]
-
+- Question (lsun20-stat6250): Why we need to use quotation mark in this statement?
 
 
 [Course Textbook Chapter 5, Problem 2]
-
+- Question (lsun20-stat6250): Is there any other statement like FILENAME statement are global?
 
 
 [Course Textbook Chapter 5, Problem 6]
+- Question (lsun20-stat6250): What is the usage of the dollar sign($)?
 
-
+- Answer (lsun20-stat6250): The dollar sign($) is used to identify the character variables.
 
 [Course Textbook Chapter 5, Problem 7]
+- Question (lsun20-stat6250): Is there any order of input these column?
 
-
+- Answer (lsun20-stat6250): We can use column input to read fields in any order.
 
 [Course Textbook Chapter 5, Problem 8]
-
+- Question (lsun20-stat6250): Can we change the statement to "income*2=income" in this situation?
```

Comment:
Ambiguous; please ask a question about concepts that doesn't depend upon knowing the Quiz Problem from the textbook.